tools/prlint/module.ts

```diff
@@ -0,0 +1,37 @@
+import * as fs from 'fs-extra';
+import * as glob from 'glob';
+import * as path from 'path';
+
+const modules: string[] = [];
+
+export function findModulePath(fuzz: string): string {
+  if (modules.length === 0) {
+    const repoRoot = path.join(__dirname, '..', '..');
+    const lernaConfig = require(path.join(repoRoot, 'lerna.json'));
+    const searchPaths: string[] = lernaConfig.packages;
+    searchPaths.forEach(p => {
+      const globMatches = glob.sync(path.join(repoRoot, p, 'package.json'));
+      const trimmed = globMatches.map(m => path.dirname(m));
+      modules.push(...trimmed);
+    });
+  }
+  
+  const regex = new RegExp(`(?<=[-_/])${fuzz}$`)
```

Comment:
since we're just using `test` and not actually using the match, can we skip the lookbehind? (regex is confusing enough)

Reply:
Of course. The lookbehind is a holdover from a previous iteration. Fixed.